src/components/Project/DnDColumnList.tsx

```diff
@@ -0,0 +1,146 @@
+import { useState, createContext } from "react";
+import styled from "styled-components";
+import { DragDropContext, Draggable, DraggableLocation, DragStart, Droppable, DropResult } from "react-beautiful-dnd";
+
+import { moveItemToAnotherList, reorderSameListItems } from "../../utils/dragAndDrop";
+
+import ColumnDefinition from "../../interfaces/Column";
+import Column from "../Column";
+
+export const GRABBED_ITEM_CATEGORY = {
+  COLUMN: "COLUMN",
+  ROW_TICKET: "ROW_TICKET",
+} as const;
+
+type GrabbedItemCategory = keyof typeof GRABBED_ITEM_CATEGORY;
+
+export const GrabbedItemContext = createContext<GrabbedItemCategory | null>(null);
```

Comment:
These three variables could be in a separate file, but I left them here as it'd just make codes spread if I make multiple files for these. I think for now it's ok for them to sit here so that we can reference what they are for quite quickly